### PR TITLE
Change charter to reflect new direction

### PR DESCRIPTION
**Pull request by jcbhmr**
Prompted by https://github.com/devcontainers-contrib/features/issues/316

Generated from old charter & prompt from ChatGPT https://sharegpt.com/c/6HyyijQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,78 +1,20 @@
...
-[@devcontainers-contrib]: https://github.com/devcontainers-contrib
-[community dev containers]: https://github.com/devcontainers-contrib
-[devcontainers/features]: https://github.com/devcontainers/features#readme
-[devcontainers/images]: https://github.com/devcontainers/images#readme
+# [Community Dev Containers](https://github.com/devcontainers-contrib) Charter
+
+The [Community Dev Containers](https://github.com/devcontainers-contrib) project
+is just a fun little side project 🤷 that I'm keeping up to document, maintain,
+and maybe even improve the official devcontainers images, features, and
+templates.
```
Comment (eitsupi) [inline README.md:6]: I find this statement confusing because this organization has nothing to do with official images, features or templates.

Comment (jcbhmr) [reply README.md]: Good point. I have removed that part. I don't know how I completely overlooked that. I'll try to make sure I proofread ChatGPT more next time.

**Comment by jcbhmr** [comment]
Now it's short enough to just paste here as a preview:

image

**Comment by jcbhmr** [comment]
@danielbraun89 📬

**Pull request closed without merging**